src/Simplex/Chat/Controller.hs

```diff
@@ -363,6 +363,9 @@ data ChatCommand
   | APIRemoveMembers {groupId :: GroupId, groupMemberIds :: Set GroupMemberId, withMessages :: Bool}
   | APILeaveGroup GroupId
   | APIListMembers GroupId
+  | APIMemberSupportChats GroupId
```

Comment:
I disagree with this being a separate API to list these chats and not just return the list of members, where the presence of support chat and any statistics is indicated by some object.

The argument that there are fewer supports chats than members is invalid as eventually we'd need to support pagination with search, sorting and filtering anyway, so the presence of support chat is just another filter.

Reply:
maybe